packages/perseus-core/src/widgets/input-number/input-number-util.ts

```diff
@@ -0,0 +1,29 @@
+import type {PerseusInputNumberWidgetOptions} from "../../data-schema";
+
+/**
+ * For details on the individual options, see the
+ * PerseusInputNumberWidgetOptions type
+ */
+type InputNumberPublicWidgetOptions = Pick<
+    PerseusInputNumberWidgetOptions,
+    | "answerType"
+    | "inexact"
+    | "maxError"
+    | "rightAlign"
+    | "simplify"
+    | "size"
+    | "customKeypad"
+>;
+
+/**
+ * Given a PerseusInputNumberWidgetOptions object, return a new object with only
+ * the public options that should be exposed to the client.
+ */
+function getInputNumberPublicWidgetOptions(
```

Comment:
For some reason this didn't exist, so I made one.

Reply:
I think we (or I) was under the assumption at the time that all input number would be replaced with numeric input. I think that's not how it turned out, though I know some work was done towards that goal. Thanks for adding these!